### DISCUSSION TITLE
Date/Time Format

### DISCUSSION THREAD

**Discussion by rikardronnkvist** [jheredianet/Teslamate-CustomGrafanaDashboards#15] [General]
Would suggest that you use dateTimeAsSystem instead of dateTimeAsLocal for date and time

Something like this: adriankumpf/teslamate#3053

For example the dashboard Current State have a last state change where the dateformat is based on the operating system instead of what I have specified for the Grafana-container.

**Comment by jheredianet** [comment]
I see what you mean. Interesting point. I've created as an [issue](https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/issues/16) from your suggestion.
